UnitsNet/CustomCode/UnitValueAbbreviationLookup.cs

```diff
@@ -63,6 +63,7 @@ internal List<string> GetAbbreviationsForUnit(int unit)
 
         internal List<int> GetUnitsForAbbreviation(string abbreviation)
         {
+            abbreviation = abbreviation.ToLower();
```

Comment:
I would make this a separate variable `lowercaseAbbreviation` to make it more clear. You can choose the name, but make the variable `lower` in the Add method below the same name for consistency.